spring-cloud-gcp-data-datastore/src/test/resources/index.yaml

```diff
@@ -2,8 +2,11 @@ indexes:
 
 - kind: test_entities_ci
   properties:
-  - name: embeddedEntity.stringField
-  - name: blobField
+  - name: size
   - name: color
-  - name: shape
-  - name: size
\ No newline at end of file
+
+- kind: test_entities_ci
```

Comment:
this is a very minor quibble, and I don't even know if it's worth changing, but the index might not be used from CI. I just applied it locally to test.

Reply:
Interesting, when testing on my gcp project, it would fail with this error below if not setting this index.
```
com.google.cloud.datastore.DatastoreException: no matching index found. recommended index is:
- kind: test_entities_ci
  properties:
  - name: color
  - name: __key__
    direction: desc
```

Reply:
Right, on mine too. Sorry, I thought you've introduced this "kind", but I see it's been there forever. Ignore my comment.